app/routes/_auth+/reset-password.tsx

```diff
@@ -16,29 +17,49 @@ import { GeneralErrorBoundary } from '~/components/error-boundary'
 import { authenticator, resetUserPassword } from '~/utils/auth.server'
 import {
 	Button,
+	ErrorList,
 	Field,
-	getFieldsFromSchema,
-	preprocessFormData,
-	useForm,
```

Comment:
Let's also delete these functions from the `app/utils/forms.tsx` file as they are not needed anymore!

Reply:
Haven't moved all the forms to use conform. So we might have to keep them around for just a little bit longer